src/components/common/Extensions/UserHub/partials/NotificationsTab/partials/Notification/Notification.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { type FC } from 'react';
+
+import { useGetFullColonyByAddressQuery } from '~gql';
+import {
+  NotificationType,
+  type Notification as NotificationInterface,
+} from '~types/notifications.ts';
+
+import ActionNotification from './Action/ActionNotification.tsx';
+import ExpenditureNotification from './Expenditure/ExpenditureNotification.tsx';
+
+const displayName = 'common.Extensions.UserHub.partials.Notification';
+
+interface NotificationProps {
+  notification: NotificationInterface;
+}
+
+const Notification: FC<NotificationProps> = ({ notification }) => {
+  const { data: colonyData, loading: loadingColony } =
+    useGetFullColonyByAddressQuery({
+      variables: {
+        address: notification.customAttributes?.colonyAddress || '',
+      },
+      skip: !notification.customAttributes?.colonyAddress,
```

Comment:
Should we skip also if there is no `notification.customAttributes?.notificationType`?